### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dioptre**
Added a useful example (I got so lost with the lack of documentation).

### CODE REVIEW

**Comment by dioptre** [comment]
It would be nice if you had anything. Nothing is not ideal :/ 

I'll leave it with you.

Thanks

**Comment by petermattis** [comment]
> Nothing is not ideal 

Are you aware of the [Pebble Go docs](https://godoc.org/github.com/cockroachdb/pebble#DB)? Should probably add a link to those docs in the README. There is a standard badge somewhere for doing so.

**Comment by dioptre** [comment]
No I didn't see it. Thanks.

With all due respect though, that's nowhere to start for most people who learn by example or just want to get started. I think a simple real-world example would really help. You are welcome to keep mine/Lei's until you find something better - but I really think this or a replacement (exactly what you might envision) is needed.

**Comment by dioptre** [comment]
Could you leave it in and replace it at a later date? That might be an option. I feel the concrete wall to get started will melt away with something like this.

**Comment by petermattis** [comment]
> With all due respect though, that's nowhere to start for most people who learn by example or just want to get started. I think a simple real-world example would really help. You are welcome to keep mine/Lei's until you find something better - but I really think this or a replacement (exactly what you might envision) is needed.

I don't disagree about the utility of a simple example of using Pebble. I'm pushing back on this PR because it doesn't show an example of using Pebble. Including an example that uses another library seems very strange to me. Am I completely missing something about your argument?

**Comment by dioptre** [comment]
What's your issue exactly? The example provides an in production example of how to open/create/select/iterate/etc. over a pebble database without any dependencies.

**Comment by petermattis** [comment]
> What's your issue exactly? The example provides an in production example of how to open/create/select/iterate/etc. over a pebble database without any dependencies.

I think we must be talking past each other here. The example code you've added to the README uses a `NewKVStore` method. There is no such method in the Pebble code base. A pure Pebble example would use `pebble.Open`.

**Comment by dioptre** [comment]
Ah maybe you didn't see the link to the file above that: https://github.com/sfproductlabs/tracker/blob/189ef2ea909bc0d57f258332d8b9d79e22f24683/pkv.go#L194-L266 ?

**Comment by petermattis** [comment]
@dioptre Yes, I saw the link. I'm a bit befuddled about how we're miscommunicating. Let me try again. The example code you've added uses `github.com/sfproductlabs/tracker`. It doesn't actually show any Pebble code! That seems problematic to me as the reader is looking for an example of Pebble code. I'd replace your example with something like the following:

```
package main

import (
	"fmt"
	"log"

	"github.com/cockroachdb/pebble"
)

func main() {
	db, err := pebble.Open("demo", &pebble.Options{})
	if err != nil {
		log.Fatal(err)
	}
	key := []byte("hello")
	if err := db.Set(key, []byte("world"), pebble.Sync); err != nil {
		log.Fatal(err)
	}
	value, closer, err := db.Get(key)
	if err != nil {
		log.Fatal(err)
	}
	fmt.Printf("%s %s\n", key, value)
	if err := closer.Close(); err != nil {
		log.Fatal(err)
	}
	if err := db.Close(); err != nil {
		log.Fatal(err)
	}
}
```

**Comment by dioptre** [comment]
Ok mate, I think we're getting there. I hope this works.

**Comment by dioptre** [comment]
I can upload the file to your repo? I like what you started, but I think a complete example would be so valuable for people. I know you think just using the raw code is useful, but having a production use case would be such a benefit to users, especially in getting a full grasp for what pebble can offer.

Also you can squash the pr when you'd like (and you're happy with the commits) as per the below image:

Screen Shot 2020-08-13 at 11 54 58 AM

**Comment by petermattis** [comment]
> I can upload the file to your repo? I like what you started, but I think a complete example would be so valuable for people.

Adding that file to the Pebble repo is not what I had in mind. That code is part of a larger code base. The example should be isolated/independent/standalone. Something that could be copied into a file a run. If you want a larger example showing various functionality, then please write one. Or file an issue.

**Comment by dioptre** [comment]
Ok I've omitted the line.

**Comment by petermattis** [comment]
LGTM. Thanks for the contribution!

**Comment by SWDevAngel** [comment]
@dioptre LAST DAY TO ORDER!
Hi Andrew. Thank you for contributing to CockroachDB this year. As a token of our appreciation we would like to send you a gift. We tried to reach you via email, but if you didn’t see the message please DM me on our [community Slack](https://cockroa.ch/slack) `@lisa` so I can send you a link to order your gift. (If you don’t want a gift, we also have a charitable contribution choice.) All orders must be in by December 13 (that’s tomorrow!), so please send this asap. :)
